modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -105,30 +105,37 @@ def fetch_cached_or_service(validated_params)
         # otherwise fetch appointments from the upstream service
         if appointments
           Rails.logger.info('mobile appointments cache fetch', user_uuid: @current_user.uuid)
+          Mobile::V0::Appointment.set_cached(@current_user, appointments)
+          filter_pending(appointments, validated_params)
```

Comment:
Only setting cache for VAOS v0 flow and when appointment were fetched from the cache (second case seems redundant but assuming this was originally done to re-up the expiration on the cache. ). 

This is done so we don't cache paginated results. soon we'll support pre-caching for VAOS v2 so this shouldn't be an issue.

Reply:
I'm a little confused by your statement ` this was originally done`. The only cache setting i see in the existing code is in the else block, so I think this is new behavior. I don't think it's particularly problematic behavior because we bust the cache on changes, so the only way it would become a problem is if the user made a change on web. But I think it's still a change.

Reply:
yup. I was confusing the filtering of pending appts and the caching. Just decided to revert back to the original filtering logic since it's less redundant. just for the case of VAOS v2, that filter won't do anything since it's already happening in the upstream service call.